### PR TITLE
Fix: Closed incomplete bracket in selling_controller

### PR DESCRIPTION
**Pull request by hitenshKharva**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

**Review by rvpasari — APPROVED** [review]
Looks good.

File: erpnext/controllers/selling_controller.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def set_total_in_words(self):
 		from frappe.utils import money_in_words
 
 		if self.meta.get_field("base_in_words"):
-			base_amount = abs(self.base_grand_total
+			base_amount = abs(self.base_grand_total)
```
Comment (hrwX) [inline erpnext/controllers/selling_controller.py:118]: - This is not an issue, if you see its a single line `if-else` condition.
- Can you check the error traceback again what this issue is

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/controllers/selling_controller.py (hunk 1 above)
Comment (hitenshKharva) [reply erpnext/controllers/selling_controller.py:118]: Sure will check the error traceback again

**Comment by rvpasari** [comment]
TypeError: bad operand type for abs(): 'NoneType'
  File "frappe/app.py", line 61, in application
    response = frappe.handler.handle()
  File "frappe/handler.py", line 24, in handle
    data = execute_cmd(cmd)
  File "frappe/handler.py", line 63, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "__init__.py", line 1055, in call
    return fn(*args, **newargs)
  File "erpnext/shopping_cart/cart.py", line 61, in place_order
    quotation.submit()
  File "frappe/model/document.py", line 867, in submit
    self._submit()
  File "frappe/model/document.py", line 856, in _submit
    self.save()
  File "frappe/model/document.py", line 273, in save
    return self._save(*args, **kwargs)
  File "frappe/model/document.py", line 296, in _save
    self.insert()
  File "frappe/model/document.py", line 230, in insert
    self.run_before_save_methods()
  File "frappe/model/document.py", line 900, in run_before_save_methods
    self.run_method("validate")
  File "frappe/model/document.py", line 797, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "frappe/model/document.py", line 1073, in composer
    return composed(self, method, *args, **kwargs)
  File "frappe/model/document.py", line 1056, in runner
    add_to_return_value(self, fn(self, *args, **kwargs))
  File "frappe/model/document.py", line 791, in <lambda>
    fn = lambda self, *args, **kwargs: getattr(self, method)(*args, **kwargs)
  File "erpnext/selling/doctype/quotation/quotation.py", line 26, in validate
    super(Quotation, self).validate()
  File "erpnext/controllers/selling_controller.py", line 39, in validate
    super(SellingController, self).validate()
  File "erpnext/controllers/stock_controller.py", line 25, in validate
    super(StockController, self).validate()
  File "erpnext/controllers/accounts_controller.py", line 82, in validate
    self.set_total_in_words()
  File "erpnext/controllers/selling_controller.py", line 119, in set_total_in_words
    if self.is_rounded_total_disabled() else self.base_rounded_total)

**Pull request closed without merging**